docker/docker-compose.yml

```
@@ -0,0 +1,54 @@
+      - spotease-data:/var/lib/postgresql/data
+      - ./init-db.sql:/docker-entrypoint-initdb.d/init-db.sql
+    networks:
+      - spotease-network
+
+  redis:
+    image: redis:7.4.1-alpine
+    container_name: redis
+    ports:
+      - 6379:6379
+    networks:
+    - spotease-network
```

yamllint

[warning] wrong indentation: expected 6 but found 4

(indentation)